spec/factories/employees.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :employee do
+    email { Faker::Internet.email }
+    encrypted_password { Faker::Internet.password }
```

Comment:
Think about using `sequence` to create unique emails. This way you will not be limited in creating employees by the factory.

Reply:
You can also use `unique` method like:
`email { Faker::Internet.unique.email }`
but according to the documentation it can slow down your tests, so in general it is better to use sequence - just like Michał has mentioned

Reply:
Sequence used - thanks!

Reply:
why not `password { 'password' }`? Using faker cost time - why waste it on setting a password you wont use anyway?

Reply:
Simplified. Thanks!